src/debugger.ts

```diff
@@ -0,0 +1,245 @@
+import path from "path";
+import fs from "fs";
+import { ChildProcessWithoutNullStreams, spawn } from "child_process";
+
+import {
+  DebugAdapterDescriptorFactory,
+  DebugConfigurationProvider,
+  DebugSession,
+  DebugAdapterExecutable,
+  DebugAdapterDescriptor,
+  WorkspaceFolder,
+  CancellationToken,
+  DebugConfiguration,
+  ProviderResult,
+  workspace,
+  DebugAdapterNamedPipeServer,
+  debug,
+  ExtensionContext,
+  Disposable,
+  window,
+} from "vscode";
```

Comment:
We've so far been explicitly namespacing classes under the `vscode` namespace, and I think I prefer that since it makes it easier to see what types are coming from VSCode. Should we continue doing the same here?

Reply:
Yeah, agreed.